typescript-json-schema.ts

```diff
@@ -1677,12 +1688,28 @@ export function exec(filePattern: string, fullTypeName: string, args = getDefaul
 
     const json = stringify(definition, { space: 4 }) + "\n\n";
     if (args.out) {
-        require("fs").writeFile(args.out, json, function (err: Error) {
-            if (err) {
-                throw new Error("Unable to write output file: " + err.message);
+        const fs = require("fs");
+        fs.writeFile(args.out, json, function (err: Error) {
```

Comment:
Let's simplify this code and always run `fs.mkdir`. It should be no-op when the dir already exists.